test/UniswapV2PairTest.sol

```diff
@@ -13,6 +13,13 @@ contract UniswapV2PairTest is Test {
     IERC20 private token0;
     IERC20 private token1;
 
+    uint112 constant  MINIMUM_LIQUIDITY =10 ** 3;
```

Comment:
We should see if we can find an auto formatter to take care of missing or extra spaces. Until then, I think we should try to be consistent and aware of spacing. A few lines need an extra space or two. 

- This line between `=10`.
- line 11 `from,address`
- 58 `expectedLiquidity-MINIMUM_LIQUIDITY`

 PR looks great, just these two cosmetic suggestions.

Reply:
I was heavily relying on the auto formatter with LiteIde when coding Go. I will find a one for Atom.